src/server/pkg/ppsutil/decoder.go

```diff
@@ -107,7 +107,7 @@ func (r *PipelineManifestReader) NextCreatePipelineRequest() (*ppsclient.CreateP
 	case err == io.EOF:
 		return nil, err
```

Comment:
I think that's probably true in general. `io.EOF` is a weird case though, because it's only kind of an error. We should move to using `errors.Is` in place of direct comparison to `io.EOF`, but we can't count on external libraries to do the same thing, `err == io.EOF` is still the official way to check for EOF I think. So I think we should be careful about wrapping `io.EOF`s.